app/src/main/java/org/mozilla/rocket/content/ContentPortalView.kt

```diff
@@ -97,10 +98,11 @@ class ContentPortalView : CoordinatorLayout, ContentAdapter.ContentPanelListener
         if (visibility == VISIBLE) {
             return
         }
-
         HomeFragmentViewState.lastOpenNews()
         contentPortalListener?.onShow()
 
+        TelemetryWrapper.openLifeFeedNews()
```

Comment:
It's actually opening the content portal, consider renaming this with Eddie.

Reply:
fixed.